@xen-orchestra/lite/src/components/PanelHeader.tsx

```diff
@@ -37,8 +37,8 @@ interface Props extends React.ButtonHTMLAttributes<HTMLButtonElement> {
 const PanelHeader = withState<State, Props, Effects, Computed, ParentState, ParentEffects>(
   {},
   ({ actions = [], titleProps = {}, buttonGroupProps = {}, children = null }) => (
-    <Stack direction='row' alignItems='start'>
-      <Typography variant='h1' sx={DEFAULT_TITLE_STYLE} {...titleProps}>
+    <Stack direction='row' alignItems='start' sx={{ paddingTop: '1em', background: '#FFFFFF' }}>
```

Comment:
IMO, we have to avoid hardcoding hexa codes as props. It can make the theming difficulte